setup.py

```diff
@@ -14,6 +14,6 @@
     packages=["cidc_api.config"],
     py_modules=["cidc_api.models", "cidc_api.gcloud_client", "cidc_api.emails"],
     url="https://github.com/CIMAC-CIDC/cidc_api-gae",
-    version="0.9.4",
+    version="0.9.5",
```

Comment:
Tip: I'm generally trying to line up api major.minor versions with schemas. Makes it easier to track.